package.json

```diff
@@ -64,12 +64,15 @@
     "eslint-plugin-es5": "^1.5.0",
     "husky": "^4.2.5",
     "mongodb": "^3.6.2",
-    "tsdx": "^0.13.2",
-    "tslib": "^2.0.0",
-    "typescript": "^3.9.6"
+    "tsdx": "^0.14.1",
+    "typescript": "^4.1.3"
   },
   "dependencies": {
     "lodash": "^4.17.20",
     "lodash-es": "^4.17.15"
+  },
+  "resolutions": {
+    "**/@typescript-eslint/eslint-plugin": "^4.1.1",
+    "**/@typescript-eslint/parser": "^4.1.1"
```

Comment:
https://stackoverflow.com/a/63919395/590396

Reply:
I'd like to stay on the old version of `tsdx` until this workaround isn't required any longer.

Or are there any new features in TSC 4.1 that SuperJSON would profit from?

Reply:
Fair enough